src/plugins/views/Tabs/hooks/useTabLayout.js

```diff
@@ -538,6 +542,7 @@ const useTabLayout = (props, dockRef) => {
       };
 
       emit(PLUGINS.TABS.ON.ACTIVE_TAB_CHANGE, { id: tabData.id });
+      setUrl(tabData.id);
```

Comment:
Should be before emit

Reply:
Yes. Because we may not have to pass down url when adding keybinds if we do this in the correct order. Which means no useKeyBinds needed. It is good forward thinking to reposition this.